packages/headless/src/features/analytics/insight-analytics-actions.ts

```diff
@@ -33,7 +32,6 @@ export const logInsightCreateArticle = (
     (client, state) => {
       validatePayload(createArticleMetadata, {
         articleType: requiredNonEmptyString,
-        triggeredBy: requiredNonEmptyString,
```

Comment:
I see that `triggeredBy` is still included in the `analytics/expandToFullUI` action meta. Do we want to remove it from there as well?

More fundamentally, though, this change will essentially "break" customer reports using `triggeredBy` as a custom dimension. Should we instead postpone this change to UI Kit V3?

Reply:
That is a good point, I think we could leave the `triggeredBy` in the expandToFullUI until V3. Personnally, I dont see much value in this action meta.... @mmitiche @erocheleau what do you guys think? I believe the `triggeredBy` for logExpandToFullUI was added essentially for the same reasons

Reply:
Yea leave it as-is in the expandtofullui, and no this won't break customers reporting on it since this is just us creating the first time this would be logged with hosted insight panel.